testing/integration/examples/integration.rs

```diff
@@ -50,7 +51,7 @@ pub fn main() {
         .unwrap();
 
     // Instantiate machine
-    tester.instantiate_machine().unwrap();
+    tester.instantiate_machine(DummyExterns).unwrap();
```

Comment:
Maybe we could do some magic here with a default implementation for the Externs type param, but this is good enough ;-)